docs/reference/commands/keystore.asciidoc

```diff
@@ -0,0 +1,151 @@
+[role="xpack"]
```

Comment:
I know we're in a bit of a transitional state with x-pack, but the code for this tool doesn't come from the `x-pack` module. Is this labelled x-pack because it's a security feature?

Reply:
Yes, I assumed it was not in the OSS-only distribution but I'll remove that, thanks